src/parser/jobs/nin/modules/TrickAttackUsage.js

```diff
@@ -78,7 +80,7 @@ export default class TrickAttackUsage extends Module {
 			this.suggestions.add(new TieredSuggestion({
 				icon: ACTIONS.TRICK_ATTACK.icon,
 				content: <Trans id="nin.ta-usage.suggestions.opener.content">
-					Avoid unconventional timings for your first <ActionLink {...ACTIONS.TRICK_ATTACK}/> of the fight in order to line it up with all the other raid and personal buffs. In most openers, Trick Attack should be weaved in approximately 10 seconds into the fight.
+					Avoid unconventional timings for your first <ActionLink {...ACTIONS.TRICK_ATTACK}/> of the fight in order to line it up with all the other raid and personal buffs. In most openers, Trick Attack should be weaved in approximately 8-9 seconds into the fight.
```

Comment:
The suggestion implies you are checking based on time in seconds, but you are actually checking gcd counts (also see abvoe comment).  Various things like gcd drag, using tcj, or using non-2-part ninjutsus will through the timings off.

Reply:
Same as above, going to leave this for now and revisit down the line. The reason I opted for time in the suggestion text is because it felt more understandable, particularly with regards to aligning it with other raid buffs that are going to at x point in the fight. It may be worth changing to show a GCD count instead, but I'd rather have that discussion outside the scope of this PR and just have this one be for correcting existing logic without changing it significantly (at least, outside of the places where significant changes are necessary).